packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/forceFunctionCreate.test.ts

```diff
@@ -116,19 +120,34 @@ describe('Force Function Create', () => {
       const errorText = 'custom error text';
       execStub.yields(new Error(errorText));
       funcCreate.runPostCommandTasks(functionInfoJava).catch();
-      sinon.assert.calledOnce(execStub);
-      sinon.assert.calledWith(execStub, 'mvn install', { cwd: path.join('some', 'dir', 'functions', 'myFunc1') });
-      sinon.assert.calledWith(
+      assert.calledOnce(execStub);
+      assert.calledWith(execStub, 'mvn install', { cwd: path.join('some', 'dir', 'functions', 'myFunc1') });
+      assert.calledWith(
         notificationServiceStub,
         nls.localize('force_function_install_mvn_dependencies_error', errorText)
       );
     });
 
+    it('Log metric should add additional language property for telemetry for java function', async () => {
```

Comment:
The convention for test subjects is to start it with "Should", so `it('Should.....') {}`